rnpm-postlink.js

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env node
+
+'use strict';
+
+var fs = require('fs');
+var rnpm = require('rnpm/src/config');
+var xcode = require('xcode');
+
+// The current working directory should be project root of the app that is linking Realm.
```

Comment:
Can you remove the references to Realm here and replace with react-native-storage?